tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -137,7 +144,7 @@ public static string GetNativeLoaderPath()
             {
                 ("win", "X64")     => "Datadog.AutoInstrumentation.NativeLoader.x64.dll",
                 ("win", "X86")     => "Datadog.AutoInstrumentation.NativeLoader.x86.dll",
-                ("linux", "X64")   => "Datadog.AutoInstrumentation.NativeLoader.so",
+                ("linux", "X64")   => "Datadog.Trace.ClrProfiler.Native.so",
```

Comment:
Ok, seeing this list is making me question everything around the naming stuff here. I Definitely need to find that doc 😬

Reply:
I will be glad to see that doc

Reply:
Ah right, on Linux we want to keep the configuration the same. Since this is the filename for the tracer native binary, this checks out